00_Base/src/model/DTO/CredentialsRequestDTO.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  ArrayMinSize,
+  IsArray,
+  IsNotEmpty,
+  IsString,
+  IsUrl,
+  ValidateNested,
+} from 'class-validator';
+import { Type } from 'class-transformer';
+import { CredentialsRoleDTO } from './CredentialsRoleDTO';
+
+export class CredentialsRequestDTO {
```

Comment:
may be helpful to have Admin in the name to separate from the other credential endpoints